CONTRIBUTING.md

```diff
@@ -248,3 +248,65 @@ For building the copr locally, you will need:
 ```bash
 make copr-build
 ```
+
+### Avoiding calling `yum` API in some specific cases
```

Comment:
@abadger I've added a small statement about this case in the `CONTRIBUTING.md`. Let me know if it's good enough as I don't know if I did a good job explaining it.